superset/assets/javascripts/SqlLab/components/QuerySearch.jsx

```diff
@@ -185,16 +185,20 @@ class QuerySearch extends React.PureComponent {
             Search
           </Button>
         </div>
-        <QueryTable
-          columns={[
-            'state', 'db', 'user', 'date',
-            'progress', 'rows', 'sql', 'querylink',
-          ]}
-          onUserClicked={this.onUserClicked.bind(this)}
-          onDbClicked={this.onDbClicked.bind(this)}
-          queries={this.state.queriesArray}
-          actions={this.props.actions}
-        />
+        <div className="scrollbar-container">
```

Comment:
how are you setting the height on the scrolling container? wonder if this is an opportunity to create a `<ScrollingContainer />` component since we are using it in a few places..